src/store/model/allneeds.ts

```diff
@@ -0,0 +1,64 @@
+import {Store} from './store';
+
+export const Allneeds: Store = {
+	backoffStatusCodes: [403, 429],
+	currency: '$',
+	labels: {
+		inStock: {
+			container: '.amstockstatus',
+			text: ['In Stock']
+		},
+		maxPrice: {
+			container: 'span.price',
+			euroFormat: false
+		},
+		outOfStock: {
+			container: '.amstockstatus',
+			text: ['Sold Out', 'Sold Out - Check back or email us for updates']
```

Comment:
```suggestion
			text: ['sold out']
```

should be sufficient

Reply:
included with build d33c279